pyiron_base/jobs/job/generic.py

```diff
@@ -430,6 +431,15 @@ def refresh_job_status(self):
                     self.project_hdf5.file_name, self.job_name + "/status"
                 )
             )
+        if (
+            isinstance(self.server.future, Future)
+            and not self.status.finished
+            and self.server.future.done()
+        ):
+            if self.server.future.cancelled():
+                self.status.aborted = True
+            else:
+                self.status.finished = True
```

Comment:
That works for me for now.

Reply:
I merged the changes you suggested, so if there are no more modification request, can you approve the pull request? Then we have everything clearly documented here.